doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -84,6 +84,22 @@ It sets Aggressive Link Power Management (ALPM) to the value specified by the [o
 `mounts`::
 Enables or disables barriers for mounts according to the Boolean value of the [option]`disable_barriers` option. 
 
+`scheduler`::
+Allows tuning of scheduling priorities, processes/threads/IRQs affinities, and CPU cores isolation.
++
+The `default_irq_smp_affinity` parameter controls what *Tuned* will write to the `/proc/irq/default_smp_affinity`.
```

Comment:
Suggestion:
s/what *Tuned* will write/the values *Tuned* writes/
s/to the/to/
Even though the file is "unique" I believe 'the' should be dropped as it is is "name".  A document person should review this.